komodoenv/creator.py

```diff
@@ -32,15 +30,16 @@ def __exit__(self, type, value, traceback):
         self.io.close()
 
 
-class Creator(object):
+class Creator:
     _fmt_action = "  " + green("{action:>10s}") + "    {message}"
 
-    def __init__(self, komodo_root, srcpath, dstpath=None, use_color=False):
+    def __init__(self, komodo_root, srcpath, trackpath, dstpath=None, use_color=False):
```

Comment:
What's the difference between `srcpath` and `trackpath`?

Reply:
`komodo_root`: `/prog/res/komodo`
`srcpath`: Specific release `/prog/res/komodo/2020.30.01-py36-rhel9`
`trackpath`: Tracked release that `komodoenv-update` will update to, `/prog/res/komodo/stable-py36`

I'm still not 100% sure the updating mechanism works as I intend it though.

Reply:
I'm like 80% sure the updating mechanism works based on some minor testing.